packages/flutter/lib/src/gestures/events.dart

```diff
@@ -1917,6 +1917,103 @@ class _TransformedPointerScrollInertiaCancelEvent extends _TransformedPointerEve
   PointerScrollInertiaCancelEvent transformed(Matrix4? transform) => original.transformed(transform);
 }
 
+mixin _CopyPointerScaleEvent on PointerEvent {
+  /// The scale (zoom factor) of the event.
+  double get scale;
+
+  @override
+  PointerScaleEvent copyWith({
+    Duration? timeStamp,
+    int? pointer,
+    PointerDeviceKind? kind,
+    int? device,
+    Offset? position,
+    Offset? delta,
+    int? buttons,
+    bool? obscured,
+    double? pressure,
```

Comment:
Why are some of these parameters not used?

Reply:
I guess PointerPanZoomUpdateEvent.copyWith is doing the same thing below.

Reply:
The signature has to match `PointerEvent.copyWith`, but most parameters are not relevant